### DISCUSSION TITLE
[Pre-release] v8.2.241112-preview1

### DISCUSSION THREAD

**Discussion by michael-hawker** [CommunityToolkit/Windows#557] [Announcements]
## What's Changed

You don't need to modify your TFM off of 19041 anymore! This was a problem for folks with 8.1, but has now been fixed by working with the dependency that caused this original bump, thanks everyone for the feedback! Enjoy!

### New Controls

* **SwitchPresenter 2.0** and the new _SwitchConverter_ 🦙❤️ by @michael-hawker in https://github.com/CommunityToolkit/Windows/pull/550

### Native AOT Support

* Updated to WinAppSdk 1.6, fixed AoT annotations by @Arlodotexe in https://github.com/CommunityToolkit/Windows/pull/457
* Update Microsoft.WindowsAppSDK to 1.6-preview2 by @Arlodotexe in https://github.com/CommunityToolkit/Windows/pull/492
* Update Microsoft.WindowsAppSdk to 1.6 stable by @Arlodotexe in https://github.com/CommunityToolkit/Windows/pull/503
* Fix analyzer warnings for CsWinRTAotWarningLevel 2 by @Arlodotexe in https://github.com/CommunityToolkit/Windows/pull/507
* Fixed AttachedCardShadow under NativeAoT by @Arlodotexe in https://github.com/CommunityToolkit/Windows/pull/526
* Fixed AdvancedCollectionView not working with NativeAoT by @michael-hawker in https://github.com/CommunityToolkit/Windows/pull/535
* Fixed AoT data binding in IncrementalLoadingCollectionSample by @Arlodotexe in https://github.com/CommunityToolkit/Windows/pull/531

### Bug Fixes

* Updated Microsoft.Graphics.Win2D package reference to version 1.3.0, downgrade TFM back to 19041 by @Arlodotexe in https://github.com/CommunityToolkit/Windows/pull/540
* Enabled 'AcrylicBrush' in WinUI 3 again by @Sergio0694 in https://github.com/CommunityToolkit/Windows/pull/488
* Update _toolTip even if toolTipText isn't available by @dotMorten in https://github.com/CommunityToolkit/Windows/pull/479
* chore: update typos in class properties summary by @ajpinedam in https://github.com/CommunityToolkit/Windows/pull/471

### Infrastructure

* Fixed Codespace container build failure  by @Arlodotexe in https://github.com/CommunityToolkit/Windows/pull/458
* Fixed failed NuGet validation due to webview2 reference by @michael-hawker in https://github.com/CommunityToolkit/Windows/pull/521
* Update Directory.Build.props to Version 8.2 by @michael-hawker in https://github.com/CommunityToolkit/Windows/pull/483

## New Contributors
* @Sergio0694 made their first contribution in https://github.com/CommunityToolkit/Windows/pull/488
* @dotMorten made their first contribution in https://github.com/CommunityToolkit/Windows/pull/479
* @ajpinedam made their first contribution in https://github.com/CommunityToolkit/Windows/pull/471

**Full Changelog**: https://github.com/CommunityToolkit/Windows/compare/v8.1.240916...v8.2.241112-preview1

<hr /><em>This discussion was created from the release <a href='https://github.com/CommunityToolkit/Windows/releases/tag/v8.2.241112-preview1'>[Pre-release] v8.2.241112-preview1</a>.</em>

**Comment by BlameTwo** [comment]
Do I have to add an exclusion option as the TokenizingTextBox still experiences type conversion errors in AOT mode?
